src/viam/robot/client.py

```diff
@@ -793,7 +793,7 @@ async def stop_all(self, extra: Dict[ResourceName, Dict[str, Any]] = {}):
     # LOG #
     #######
 
-    async def log(self, name: str, level: str, time: datetime, log: str, stack: str):
```

Comment:
param rename is technically a breaking change if someone is passing named arguments

you're making it match the official API, so I'm guessing this is the right call, but flagging so we can debug quickly if it causes problems down the road

Reply:
Hmm, that is a good point! Thank you for bringing it up. I'll see what the others think.

Reply:
Good call out. Let's make this change but call it out appropriately in the breaking changes channel